Makefile

```diff
@@ -0,0 +1,5 @@
+build-docs:
+	@echo "Building documentation"
+	make -C doc/ clean
+	make -C doc/ html-noplot
+	cd doc/ && make view
```

Comment:
Newline.
Does the makefile need to be commited?

Reply:
No. This is just for my dev sake rn. I'm still wrangling upstream dependencies. I.e. pywhy-graphs. I'll remove it.

Reply:
Actually jk I think it would be nice to have this. It makes the docs building a bit easier. If someone knows how to add a multi-step procedure to the poetry command `build_docs`, then maybe not tho...

Reply:
If it's something scripted like this I would generally model this under a bash file and invoke the bash file via a poetry script.